app/views/hyrax/base/_work_type.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="work-type-tag">
+  <% if Hyrax.config.display_work_icon? %>
```

Comment:
Do we need to support this conditional?  Can we just always show the work icon?

Reply:
Yeah, what's driving the need to make this configurable?

Reply:
Overzealous cruft. Originally I was thinking the icon would be in the normal config, and added the option to enable it. Thinking through it (especially using the ModelIcon) it makes more sense to remove this and just override the view if someone wants to disable it. Will remove.